R/tm_a_pca.R

```diff
@@ -171,7 +174,8 @@ tm_a_pca <- function(label = "Principal Component Analysis",
   )
 }
 
-
+# UI function for the PCA module
+# This function defines the UI elements for the PCA analysis, including plot settings and data selection.
```

Comment:
I'm not sure this is needed

Reply:
I don't see any problem with including additional details as this information is solely meant for developers who navigate the code rather than for formal documentation. but , I will remove line 178. I have kept these detailed in this module only, rest only have titles.

also I think we should also add` @noRd` and `@keyowrd internal `

Reply:
I see you already have `noRd` and `keywords internal` in other PRs, so let's have it, if you already made some work on it